Adafruit_FRAM_SPI.h

```diff
@@ -44,6 +44,7 @@ class Adafruit_FRAM_SPI {
 public:
   Adafruit_FRAM_SPI(int8_t cs, SPIClass *theSPI = &SPI);
   Adafruit_FRAM_SPI(int8_t clk, int8_t miso, int8_t mosi, int8_t cs);
+  Adafruit_FRAM_SPI(int8_t cs, uint32_t freq, SPIClass *theSPI = &SPI);
```

Comment:
I think it would be better if we piggy packed freq into the existing constructor as default parameter

```C
Adafruit_FRAM_SPI(int8_t cs, SPIClass *theSPI = &SPI, uint32_t freq=1000000);
```

Reply:
Sure, this works too. I updated the PR accordingly. Thanks!